scripts/test_all.sh

```diff
@@ -103,44 +126,51 @@ cd ../../dutchf3_patch/local
 export CUDA_VISIBLE_DEVICES=2
```

Comment:
Do we test if the test anchored to CUDA_VISIBLE_DEVICES=2 earlier is done by now? It appears the previous job may be running in the background when this test is being called.

Reply:
This assumes ND40 VM (or any VM with 8 GPUs) - all tests are run at the same time on different GPUs.